src/anders_mathiesen_ex/ex_04/myrand.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Anders Mathiesen'
+__email__ = 'andermat@nmbu.no'
+
+
+class LCGRand:
+    def __init__(self, seed):
```

Comment:
Please put a short docstring in the __init__ for all classes explaining what it does.